packages/client/hmi-client/src/workflow/ops/stratify-mira/tera-stratify-mira.vue

```diff
@@ -208,16 +234,89 @@ const inputChangeHandler = async () => {
 	}
 };
 
-const saveNewModel = async () => {
-	if (!model.value || !newModelName.value) return;
-	model.value.header.name = newModelName.value;
+const saveNewModel = async (modelName: string, options: Record<string, boolean>) => {
+	if (!model.value || !modelName) return;
+	model.value.header.name = modelName;
 
 	const projectResource = useProjects();
 	const modelData = await createModel(model.value);
 	const projectId = projectResource.activeProject.value?.id;
 
 	if (!modelData) return;
-	await projectResource.addAsset(AssetType.Models, modelData.id, projectId);
+
+	if (options.addToProject) {
+		await projectResource.addAsset(AssetType.Models, modelData.id, projectId);
+	}
+
+	if (options.appendOutputPort) {
+		await appendOutputPort(modelData.id);
+	}
+};
+
+const appendOutputPort = async (modelId: string) => {
+	// fetch the model that was just created
+	const newModel = await getModel(modelId);
+	if (!newModel) return;
+
+	// set default configuration for the newly created model
+	await addDefaultConfiguration(newModel);
+
+	// setting timeout...elastic search might not update default config in time
+	setTimeout(async () => {
+		const configurationList = await getModelConfigurations(newModel.id);
+		configurationList.forEach((configuration) => {
+			emit('append-output-port', {
+				id: uuidv4(),
+				label: `${newModel.header.name} - ${configuration.name}`,
+				type: 'modelConfigId',
+				value: [configuration.id]
+			});
+		});
+	}, 800);
```

Comment:
This section is not needed. The outputs should be `models` and not `model-configurations`

We still have some issues with the separation of concerns between model and model-configurations, but while that is been worked out let's remove the the entire setTimeout section for now.

Reply:
done